packages/yoga/src/List/web/LinkItem.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledLinkItem = styled.a`
+  ${({
+    theme: {
+      yoga: {
+        components: { list },
+      },
+    },
+  }) => `
+    padding:
+      ${list.listItem.padding.top}px
+      ${list.listItem.padding.right}px
+      ${list.listItem.padding.bottom}px
+      ${list.listItem.padding.left}px;
+  `}
+
+  display: inline-block;
+  width: 100%;
+  height: 100%;
+
+  text-decoration: none;
+`;
```

Comment:
In all styled components we use a CSS Ordering (we didn't explained well in the DOJO, sorry).
What do you think to learn and use in this styled ?

**This is the reference:**
https://webdesign.tutsplus.com/articles/outside-in-ordering-css-properties-by-importance--cms-21685

Reply:
And we use this callback 

```
${({
    theme: {
      yoga: {
        components: { list },
      },
    },
  }) => `
    padding:
      ${list.listItem.padding.top}px
      ${list.listItem.padding.right}px
      ${list.listItem.padding.bottom}px
      ${list.listItem.padding.left}px;
  `}
```

in the end of all rules of styled component